utilities/formatDate.ts

```
@@ -0,0 +1,80 @@
+    return '';
+  }
+
+  const _date = new Date(date);
+
+  if (isNaN(_date.getTime())) {
+    console.error('Invalid date provided:', date);
+    return '';
+  }
+
+  const year = _date.getFullYear();
+  let month = '' + (_date.getMonth() + 1);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)